pkg/utils/jwt.go

```diff
@@ -36,3 +34,29 @@ func GetStringFieldFromJWT(token string, field string) (string, error) {
 
 	return claimString, nil
 }
+
+// GetUsernameFromJWT returns the username extracted from JWT token
+func GetUsernameFromJWT(token string) string {
```

Comment:
It's good , if we could write unit test to cover this func

Reply:
sure thing ;) also moved from login.go without a change. Added test cases.